### PR TITLE
[TR] PIM-2736: Rework mary sends proposal with a comment

### PR DESCRIPTION
**Pull request by willy-ahva**
| Q                 | A
| ----------------- | ---
| Added Specs       | no
| Added Behats      | no
| Travis CI is ok   | Blue
| Changelog updated | nope

### CODE REVIEW

**Comment by grena** [comment]
GTM

File: src/Pim/Bundle/NotificationBundle/Resources/config/requirejs.yml [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-        pim/notification-list:  pimnotification/js/notification-list
-        pim/form/comment:       pimnotification/js/form/comment
+        pim/indicator:         pimnotification/js/indicator
+        pim/notifications:     pimnotification/js/notifications
+        pim/notification-list: pimnotification/js/notification-list
 
         pim/template/notification/notification:        pimnotification/templates/notification/notification.html
         pim/template/notification/notification-footer: pimnotification/templates/notification/notification-footer.html
         pim/template/notification/notification-list:   pimnotification/templates/notification/notification-list.html
-        pim/template/comment/comment:                  pimnotification/templates/comment/comment.html
```
Comment (ClementGautier) [inline src/Pim/Bundle/NotificationBundle/Resources/config/requirejs.yml]: why did we move this in the EE ?

File: src/Pim/Bundle/NotificationBundle/Resources/public/templates/comment/comment.html [hunk 2]
```diff
@@ -1,4 +0,0 @@
-<div class="notification-comment">
```
Comment (ClementGautier) [inline src/Pim/Bundle/NotificationBundle/Resources/public/templates/comment/comment.html]: why did we move this in the EE ?

**Comment by ClementGautier** [comment]
GTM

**Pull request merged**